README.md

```diff
@@ -35,9 +36,9 @@
 9. Log in to the NEAR Wallet from the CLI: `$ near login`. The browser should pop up and a NEAR Wallet should ask for a permission for adding a full access key.
 
 ### Configuration
-1. Copy `test-ethereum-config.json` to `ethereum-config.json` with `$ cp eth-custodian/scripts/json/test-ethereum-config.json eth-custodian/scripts/json/ethereum-config.json`.
+1. Go to _eth-custodian_ directory: `$ cd eth-custodian`.
 
-2. Update `ethereum-config.json` with the actual data on the addresses.
+2. (Optional) Update `scripts/json/ethereum-config.json` with the actual data on the addresses.
 
 3. Create `.env` file inside `eth-custodian` directory: `$ touch .env`.
 
```

Comment:
`RPC access can be easily gained from Alchemy.` this part should be in a new line or an optional thing.

Reply:
But it's on the new line already as well. Or what do you mean?